src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java

```diff
@@ -36,12 +36,12 @@ public ProtectionManager(CSCoreLib cscorelib) {
 		if (cscorelib.getServer().getPluginManager().isPluginEnabled("WorldGuard") && cscorelib.getServer().getPluginManager().isPluginEnabled("WorldEdit")) {
 			registerNewModule("WorldGuard", new WorldGuardProtectionModule());
 		}
-		if (cscorelib.getServer().getPluginManager().isPluginEnabled("Factions")) {
+		/*if (cscorelib.getServer().getPluginManager().isPluginEnabled("Factions")) {
```

Comment:
Why was Factions support removed?

Reply:
Because API of factions changed

Reply:
Then update it accordingly or ask someone for help on this but in general we want to keep this as consistent as possible.
And CS-CoreLib functions as a bridge between protection plugins and plugins like Slimefun.
Removing all factions protections without any warning may just be very rude to do.

Reply:
Factions jar, that you are using to compile cscorelib.

Reply:
This is a Maven project.
There is no jar...
But I can point you to the pom.xml
https://github.com/TheBusyBiscuit/CS-CoreLib/blob/master/pom.xml